src/comments/templates/comments/comments.html

```diff
@@ -27,8 +27,8 @@
         <h3 class="comment-heading">
           {% if request.user.is_staff %}
           <a href="{% url 'profiles:profile_detail' comment.user.profile.id %}">{{comment.user}}</a>
-          <small>({{comment.user.profile.preferred_full_name}}
-          Current Mark: {{comment.user.profile.mark|floatformat:0}}%)
+          <small>
+            ({{comment.user.profile.preferred_full_name}}{% if config.display_marks_calculation %} Current Mark: {{comment.user.profile.mark|floatformat:0}}%{% endif %})
```

Comment:
Combined into a single line since it creates an extra whitespace
<img width="319" alt="Screen Shot 2020-08-06 at 7 53 00 AM" src="https://user-images.githubusercontent.com/10972027/89475078-daeb5980-d7b9-11ea-8b3b-114d6534f658.png">

Reply:
There should be an `else` block that displays `profile.xp_per_course` otherwise.